thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/rootcause/impl/MetricBreakdownPipeline.java

```diff
@@ -107,9 +112,11 @@
    * @param includeDimensions dimensions to break down on (all if empty)
    * @param excludeDimensions dimensions not to break down on
    * @param k number of top-ranking elements to emit
+   * @param ignoreScore flag to include all breakdowns, even if score is zero (only set if ignoring wanted)
    */
   public MetricBreakdownPipeline(String outputName, Set<String> inputNames, MetricConfigManager metricDAO,
-      DatasetConfigManager datasetDAO, QueryCache cache, ExecutorService executor, Set<String> includeDimensions, Set<String> excludeDimensions, int k) {
+      DatasetConfigManager datasetDAO, QueryCache cache, ExecutorService executor, Set<String> includeDimensions,
+      Set<String> excludeDimensions, int k, boolean ignoreScore) {
```

Comment:
Who is calling this or rather how is this being triggered? I do not see any updates to the caller in this PR.

Reply:
The rca.yml configuration file is where we trigger and configure the framework.  From the RCA Dev guide:

"Every pipeline configuration consists of four fields: "outputName" respresents the name of the pipeline. "inputNames" is a list of pipeline names that deliver their outputs as input to the current pipeline. "className" specifies the fully qualified class name of the business logic implementing this pipeline. And finally "properties" holds a map of String-Object tuples that are passed to the pipeline constructor as configuration parameters (see "coding conventions" below). The properties map is deserialized using and ObjectMapper and can contain nested lists and maps in addition to primitive values."